### DISCUSSION TITLE
[SOLVED: Was a GH UI C+P Error][Bug? Not possible?] How to add multiple Labels with ':' and a space in the Name?

### DISCUSSION THREAD

**Discussion by MeerMusik** [community/community#4602] [Projects and Issues] [answered]
Hello World! I know this is the Discussion Section but I have no Idea where the Repository for the Issue Form Template Code is.

I am currently trying to add multiple Labels like that:
`labels: ['Type: Bug', 'Status: Not Confirmed']`

which results in adding no Label at all.

If I do
`labels: ['Type: Bug']`
it works.

And yes I know that Yaml is case-sensitive. All tested Labels have the correct Text.

Thank you in advance for any hint how to this work (if possible)! :)
Olli

**Comment by MeerMusik** [comment, marked as answer]
DOH! Nevermind! I did a Copy+Paste from the Template to the "Create New Label" - something went wrong with CTRL+A & CTRL+C - it works now. Sorry

EDIT: Really? I can not close my own posts? Why? @admin / mod: Please close

**Comment by MAN-Sendance** [comment]
So the correct solution is:

`labels: ['Type: Bug', 'Status: Not Confirmed']`

**Reply by MeerMusik** [reply]
Correct!
